wire-ios-sync-engine/Source/SessionManager/SessionManager.swift

```diff
@@ -1283,22 +1323,52 @@ public final class SessionManager: NSObject, SessionManagerType {
         journal: Journal,
         logFilesProvider: LogFilesProviding
     ) -> ZMUserSession {
+        guard let newSession = createUserSession(
+            for: account,
+            with: coreDataStack,
+            journal: journal,
+            logFilesProvider: logFilesProvider
+        ) else {
+            preconditionFailure("Unable to create session for \(account)")
+        }
+
+        finishSettingUpUserSession(
+            account: account,
+            newSession: newSession,
+            coreDataStack: coreDataStack
+        )
+
+        return newSession
```

Comment:
This method was split into 1) create the user session and 2) post configuration of the session, because if the multibackend flag is on then the user session loader creates the session but still needs this post configuration.